src/definition/accessors/ILivechatRead.ts

```diff
@@ -3,7 +3,11 @@ import { ILivechatRoom } from '../livechat/ILivechatRoom';
 import { IVisitor } from '../livechat/IVisitor';
 
 export interface ILivechatRead {
-    isOnline(): boolean;
+    /**
+     * Gets online status of the livechat.
+     * @param department (optional) the id of the livechat department
+     */
+    isOnline(department?: string): boolean;
```

Comment:
@lolimay Maybe `departmentId` instead of `department` would be better.
What do you think?

Reply:
I agree, `departmentId` as used below would be the correct name.

Reply:
Done, thanks!